easybuild/easyblocks/s/suitesparse.py

```diff
@@ -162,13 +172,21 @@ def install_step(self):
     def make_module_req_guess(self):
         """
         Extra path to consider for module file:
-        * add config dir to $CPATH so include files are found
-        * add UMFPACK and AMD library dirs to $LD_LIBRARY_PATH
+        * add config dir and include to $CPATH so include files are found
+        * add UMFPACK and AMD library, and lib dirs to $LD_LIBRARY_PATH
         """
+        # Latest version of SuiteSparse put shared libraries in 'lib'
+        if LooseVersion(self.version) >= LooseVersion('4.5'):
+            ld_library_path = ['lib']
+            include_dirs = ['include']
+        else:
+            ld_library_path = ['UMFPACK/Lib', 'AMD/Lib']
+            include_dirs = [self.config_name]
+
         guesses = super(EB_SuiteSparse, self).make_module_req_guess()
         guesses.update({
-            'CPATH': [self.config_name],
-            'LD_LIBRARY_PATH': ['UMFPACK/Lib', 'AMD/Lib'],
+            'CPATH': include_dirs,
+            'LD_LIBRARY_PATH': ld_library_path,
```

Comment:
@besserox please also consider the change request in #882 when tackling the remark above